docs/language/src/builtins/builtin_elements.md

```diff
@@ -102,90 +119,140 @@ export component Example inherits Window {
     width: 270px;
     height: 100px;
 
-    Rectangle {
-        x: 10px;
-        y: 10px;
-        width: 50px;
-        height: 50px;
-        background: blue;
+    Flickable {
+        viewport-height: 300px;
+        Text {
+            x:0;
+            y: 150px;
+            text: "This is some text that you have to scroll to see";
+        }
     }
+}
+```
 
-    // Rectangle with a border
-    Rectangle {
-        x: 70px;
-        y: 10px;
-        width: 50px;
-        height: 50px;
-        background: green;
-        border-width: 2px;
-        border-color: red;
+## `FocusScope`
+
+The `FocusScope` exposes callbacks to intercept key events. Note that `FocusScope`
+will only trigger when it `has-focus`.
```

Comment:
It's unclear what "trigger" means. Let's be specific: invoke is the term we use in the context of callbacks and with "them" we're referring to them explicitly.

```suggestion
will only invoke them when it `has-focus`.
```